lib/resources/r4/condition.yaml

```diff
@@ -0,0 +1,119 @@
+---
+name: Condition
+field_name_base_url: https://hl7.org/fhir/r4/condition-definitions.html#Condition
+fields:
+- name: clinicalStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: active | recurrence | relapse | inactive | remission | resolved
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.clinicalStatus
+  binding:
+    description: The clinical status of the condition.
+    terminology:
+      - display: Condition Clinical Status Codes
+        system: https://terminology.hl7.org/CodeSystem/condition-clinical
+        info_link: https://hl7.org/fhir/r4/valueset-condition-clinical.html
+
+- name: verificationStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: unconfirmed | provisional | differential | confirmed | refuted | entered-in-error
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.verificationStatus
+  binding:
+    description: The verification status to support or decline the clinical status of the condition or diagnosis.
+    terminology:
+      - display: Condition Verification Status Codes
+        system: https://terminology.hl7.org/CodeSystem/condition-ver-status
+        info_link: https://hl7.org/fhir/r4/valueset-condition-ver-status.html
+
+- name: category
+  required: 'No'
+  cardinality: 0..*
+  type: CodeableConcept
+  description: problem-list-item | encounter-diagnosis
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.category
+  example: |
+    {
+      "category": {
+        "coding": [
+          {
+            "system": "https://terminology.hl7.org/CodeSystem/condition-category",
+            "code": "problem-list-item",
+            "display": "Problem List Item"
+          }
+        ],
+        "text": "Diagnosis"
+      }
+    }
+  binding:
+    description: A category assigned to the condition.
+    note: Category code `problem-list-item`is supported for search functionality.
+    terminology:
+      - display: Condition Category Codes
+        system: https://terminology.hl7.org/CodeSystem/condition-category
+        info_link: https://hl7.org/fhir/r4/valueset-condition-category.html
+
+- name: severity
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Severe | Moderate | Mild
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.severity
+  example: |
+    {
+      "severity": [
+        {
+          "coding": [
+            {
+              "system": ""https://snomed.info/sct",
```

Comment:
we need to be careful of the system here. snomed should be http://

also there's a double quotes at the start of the system.

Reply:
Fixed. Throughout. [diff](https://github.com/cerner/fhir.cerner.com/pull/273/commits/87ed58e3a827f1d8f8be7ceace048f199170fc55#diff-abb99d2139d79f8a2a8f1d214f5f6882R15)